app/models/rubric_criterion.rb

```diff
@@ -38,6 +39,12 @@ def validate_max_mark
     errors.add(:max_mark, 'Max mark of rubric criterion should not be greater than max level mark')
   end
 
+  def only_update_if_results_unreleased
```

Comment:
Change the name of the method. Look at how other validation method underneath this one is named, and follow the same idea.